CMakeLists.txt

```diff
@@ -1744,9 +1744,17 @@ set(CONFIG_DIR "${CMAKE_INSTALL_PREFIX}/etc/netdata")
 set(LIBCONFIG_DIR "${CMAKE_INSTALL_PREFIX}/usr/lib/netdata/conf.d")
 set(LOG_DIR "${CMAKE_INSTALL_PREFIX}/var/log/netdata")
 set(PLUGINS_DIR "${CMAKE_INSTALL_PREFIX}/usr/libexec/netdata/plugins.d")
-set(WEB_DIR "${CMAKE_INSTALL_PREFIX}/usr/share/netdata/web")
 set(VARLIB_DIR "${CMAKE_INSTALL_PREFIX}/var/lib/netdata")
 
+if(NOT DEFINED WEB_DIR)
+  set(WEB_DIR "usr/share/netdata/web")
+else()
+  string(REGEX REPLACE "^/" "" WEB_DIR "${WEB_DIR}")
+endif()
+set(WEB_DEST "${WEB_DIR}")
+set(WEB_DIR "${CMAKE_INSTALL_PREFIX}/${WEB_DEST}")
```

Comment:
Not tested by I think this is equivalent:

```
if(NOT WEB_DIR)
  set(WEB_DIR "${CMAKE_INSTALL_PREFIX}/usr/share/netdata/web")
else()
  string(REGEX REPLACE "^/" "" WEB_DIR "${WEB_DIR}")
endif()
```

I want us to ensure that **all** variables follow a common naming convention, ie. `*_DIR`. IOW, the final directory should be placed in `WEB_DIR` and reference that variable throughout the rest of the CMake code.

Reply:
Doesn't look equivalent to me. 

I don't use `WEB_DIR` because:
 - WEB_DIR should be an absolute path (starts with `/`)
 - destination should be a relative path (relative to install prefix)